CONTRIBUTING.md

```diff
@@ -1,5 +1,56 @@
-# Contribution Guide
+# How to Contribute to CodecademyCommunity.github.io
 
-This page is in the very early stages. We need designers and PMs, so let us know on Discord if you want to be a part of that!
-If you have a page or feature request, drop it in our Discussions so we can talk about adding it to the site!
-As more issues eventually get created, feel free to try your hand at implementing one and submit a PR 🙂
+All contributions are welcome! Below are some of the basic steps to contribute to our site. Please read them in their entirety to ensure you stay consistent with everyone else.
+
+Please review the [issues](https://github.com/CodecademyCommunity/CodecademyCommunity.github.io/issues) tag and self-assign one if you would like to work on it. If you are not a member of our team and therefore cannot self-assign feel free to leave a comment requesting to be assigned. This keeps the efforts tidy and helps us know who is working on what.
+
+This project is still in it's (fairly) early stages and we need designers, developers and PMs, feel free to reach out on Discord (via ModMail) if you would like to be a part of this.
```

Comment:
- Change "it's" to "its".
- Change "reach out" to "reach out to server staff".
- Change "to be a part of this" to "to get involved".